resources/osl_mongodb.rb

```diff
@@ -0,0 +1,65 @@
+resource_name :osl_mongodb
+provides :osl_mongodb, platform_family: 'rhel'
+unified_mode true
+
+default_action :install
+
+property :version, String, name_property: true
+property :install_selinux_policy, [true, false], default: true
```

Comment:
For now, let's remove this and not worry about setting up SELinux on this host. We can get back to this eventually.